exercise5/src/Controller/ValidationController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace src\Controller;
+
+use src\Model\ValidationModel;
+
+class ValidationController {
+    
+    private $error_message;
+    private $sucLogin;
+
+    public function validation()
+    {
+        if ($_SERVER['REQUEST_METHOD'] == "POST") 
+        {
+            $username = $_POST["uname"];
+            $password = $_POST["pass"];
+
+            $credentials = new ValidationModel("erikafuerte", "123456789");
+
+            if($username === $credentials->getUsername() && $password === $credentials->getPassword()) 
+            {
+                $this->sucLogin = "Welcome " . $credentials->getUsername() . "!!!<br> You are successfully logged in";
+            } elseif (empty($_POST['uname'])) {
```

Comment:
It's best practice to make your code readable. Avoid nested if statements UNLESS necessary.